test/index.js

```diff
@@ -1253,3 +1264,97 @@ describe('def', function() {
   });
 
 });
+
+describe('has long error messages', function() {
+
+  //  hasMethods :: [String] -> a -> Boolean
+  var hasMethods = R.curry(function(names, x) {
+    return x != null &&
+           R.all(function(k) { return typeof x[k] === 'function'; }, names);
+  });
+
+  var Semigroup =
+  $.TypeClass('my-package/Semigroup', hasMethods(['concat']));
+
+  var def = $.create($.env.concat([$.Nullable, Either]));
+
+  var expectedError = function(basename) {
+    var path = __dirname + '/error-messages/' + basename + '.txt';
+    return fs.readFileSync(path).toString();
+  };
```

Comment:
Not sure. I like them, except having to come up with a name for the files :pensive:  Do you have a vim plugin that auto formats those multi line string concatenations? Always frustrating for me when something on the first line changes and it becomes a cascading game of wack a mole. I guess since these are broken up over multiple lines that actually wouldn't happen. What do you think?